dtl-lexer/src/variable.rs

```diff
@@ -94,10 +94,12 @@ pub enum VariableLexerError {
     },
 }
 
-pub fn lex_variable(
+type At = (usize, usize);
```

Comment:
Should we plan to reuse this type alias in the rest of the crate ?
I'm also wondering if it would make sense to have an helper to convert from this to `miette::SourceSpan` for error reporting ?

Reply:
Yeah, I had this idea in the back of my mind. For the conversion to `SourceSpan`, do you have anything in mind other than `.into()`, which already works?